terraform/modules/bosh_vpc/network_private.tf

```diff
@@ -100,3 +98,13 @@ resource "aws_instance" "az2_private_nat" {
     Name = "${var.stack_description}  (AZ2 NAT)"
   }
 }
+
+resource "aws_eip" "az1_nat_eip" {
```

Comment:
This is both unrelated and unimportant, but I have style questions:
* Can we standardize on where `az1` goes in resource names--prefix or suffix?
* Same question for including resource types in resource names--since resources are namespaced by type, we could call this one `nat_az1` without ambiguity--but maybe it's clearer to keep the type in the name.